onchain/rollups/test/foundry/consensus/authority/AuthorityFactory.t.sol

```diff
@@ -0,0 +1,91 @@
+// Copyright Cartesi Pte. Ltd.
+
+// SPDX-License-Identifier: Apache-2.0
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+/// @title Cartesi Authority Factory Test
```

Comment:
This could be misinterpreted as the Authority controlled by some central Cartesi organization.
To avoid this misinterpretation, let us call it simply _Authority_.

Reply:
This change can be applied by merging #65.